benchmark/kernel/main_performance.py

```diff
@@ -56,21 +61,68 @@
                             shuffle=False)
     test_loader = DataLoader(test_dataset, batch_size=args.batch_size,
                              shuffle=False)
-
-    for num_layers, hidden in product(layers, hiddens):
-        print(f'--\n{dataset_name} - {Net.__name__} - {num_layers} - {hidden}')
-
-        model = Net(dataset, num_layers, hidden).to(device)
-        optimizer = torch.optim.Adam(model.parameters(), lr=args.lr)
-
-        stats_list = []
-        for epoch in range(1, args.epochs + 1):
-            loss, stats = train(model, optimizer, train_loader)
-            val_acc, val_time = eval_acc(model, val_loader)
-            test_acc, test_time = eval_acc(model, test_loader)
-
-            if epoch >= args.warmup_profile:
-                stats_list.append(stats)
-
-        stats_summary = get_stats_summary(stats_list)
-        print(stats_summary)
+    return dataset, train_loader, val_loader, test_loader
+
+
+def run_train():
+    for dataset_name, Net in product(datasets, nets):
+        dataset, train_loader, val_loader, test_loader = prepare_dataloader(
+            dataset_name)
+
+        for num_layers, hidden in product(layers, hiddens):
+            print("--\n{} - {} - {} - {}".format(dataset_name, Net.__name__,
```

Comment:
Any reason the `f`-string syntax was dropped here?
```python
print(f'--\n{dataset_name} - {Net.__name__} - '
         f'{num_layers} - {hidden}')
```

Reply:
Change back to the original one. I don't remember why it changes here.